app/src/components/navigation/TopBar.vue

```diff
@@ -2,36 +2,46 @@
 import { PlayCircleIcon, ChevronLeftIcon } from "@heroicons/vue/24/solid";
 import ProfileMenu from "./ProfileMenu.vue";
 import { useRoute, RouterLink } from "vue-router";
+import OnlineIndicator from "../OnlineIndicator.vue";
 
 const route = useRoute();
 </script>
 
 <template>
     <header class="mb-6 bg-white shadow dark:bg-zinc-700">
-        <div class="flex flex-row items-center justify-between space-x-8 px-6 py-5">
+        <div class="flex flex-row items-center justify-between space-x-2 px-6 py-5">
             <RouterLink to="/" class="flex items-center">
                 <div
-                    class="mr-4 border-r border-zinc-400 pr-4"
+                    class="mr-1 border-r border-zinc-400 pr-2"
                     :class="{
                         hidden: route.name == 'home',
                         'lg:hidden': route.name != 'home',
                     }"
                 >
                     <ChevronLeftIcon class="h-6 w-6 text-zinc-600 dark:text-zinc-50" />
                 </div>
-                <div
-                    class="flex items-center gap-2 text-xl hover:text-zinc-600 dark:hover:text-zinc-200"
-                >
+                <div class="flex items-center text-xl hover:text-zinc-600 dark:hover:text-zinc-200">
                     <PlayCircleIcon class="h-8 w-8 text-yellow-500" />
-                    <div class="hidden items-center gap-1 min-[380px]:flex">
+                    <div class="hidden items-center md:flex">
+                        <span class="font-semibold">BCC Africa</span>
+                        <span>App</span>
+                    </div>
+                    <div class="flex items-center gap-1 md:hidden">
```

Comment:
In the main branch the app name is hidden if a device has a small screen:
![image](https://github.com/user-attachments/assets/6ac8a2a2-9331-4c2e-aeb6-8c2c9e07950d)

This is however not working anymore in this branch:
![image](https://github.com/user-attachments/assets/bfae8d68-2543-48bb-9715-45ce3c224879)

The spacing between the logo and app name text also seems to have changed. It looks like this on the main branch:
![image](https://github.com/user-attachments/assets/20c5d9fe-505f-4c93-905c-7ca8239d53e9)